tests/integ/test_telemetry.py

```diff
@@ -9,6 +9,10 @@
 from snowflake.snowpark.functions import col, lit, max as max_, mean
 from tests.utils import TestData
 
+pytest.skip(
```

Comment:
check the value of `_use_sql_simplifier` and use pytest.skipif? Also for other tests you skipped. It's not good to skip existing tests when introducing a new change.